app/src/main/kotlin/com/wire/android/mapper/MessageMapper.kt

```diff
@@ -63,7 +63,17 @@ class MessageMapper @Inject constructor(
         )
     }.distinct()
 
-    fun toUIMessages(userList: List<User>, messages: List<Message.Standalone>): List<UIMessage> = messages.mapNotNull { message ->
+    fun toUIMessages(userList: List<User>, messages: List<Message.Standalone>): List<UIMessage> {
+        val uiMessages = mutableListOf<UIMessage>()
+        messages.mapNotNull { message ->
+            toUIMessage(userList, message)?.let {
+                uiMessages.add(it)
+            }
+        }
+        return uiMessages
+    }
```

Comment:
Is this function being used?

If not, we can remove.

If it is, I don't see the reason for creating a MutableList with unknown initial size manually.
```suggestion
    fun toUIMessages(userList: List<User>, messages: List<Message.Standalone>): List<UIMessage> {
        return messages.mapNotNull { message ->
            toUIMessage(userList, message)
        }
    }
```

Reply:
removed